src/sentry/static/sentry/app/utils/withTeamsForUser.tsx

```diff
@@ -45,6 +48,12 @@ const withTeamsForUser = <P extends InjectedTeamsProps>(
           teams,
           loadingTeams: false,
         });
+
+        // also fill up TeamStore and ProjectStore so org context does not have
+        // to refetch org details due to lack of teams/projects
+        const projects = _.uniqBy(_.flatten(teams.map(team => team.projects)), 'id');
+        ProjectActions.loadProjects(projects);
```

Comment:
This might be ok for this PR, but we may have to rethink it when we apply the lightweight org store since this projects set will not match what is current in prod for admins and managers (project access vs membership).

Reply:
mmm yes I see, because the hydrated organization will only have the projects from the teams that the user is in, which won't match what happens with organization details currently from the backend serializer.

Reply:
will the global selection header be a place where all accessible projects will be shown?

Reply:
Yeah, mark is doing some changes around that (adding explicit options for all projects and projects you're a member of)

Reply:
Yes, all accessible projects are currently visible in the header under the 'Projects I don't belong to' heading.

Reply:
The `organizations_teams` endpoint will give you a list of all the teams in the organization and include whether or not the current user is a member or has access to those teams. That should let you better approximate what organization details returns today.

Reply:
Gotcha, is the only place that projects by membership are shown the projects page? Would other pages such as settings -> projects and global selection show projects that are accessible by the user?

Reply:
Old discover has a warning that pops up if you try to query projects that you  are not a member of IIRC.